app.rb

```diff
@@ -0,0 +1,119 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+def list_all_book
+  puts 'You have no book in the cart!' if Book.book_list.empty?
+  Book.book_list.each do |book|
+    puts "Title: #{book.title}, Author: #{book.author}"
+  end
+  main
+end
+
+def list_all_people
+  persons = Student.student_list + Teacher.teacher_list
+  persons.length
+  puts 'optionsdfsdf'
+  persons.each do |person|
+    puts "Name: #{person.name}, Age: #{person.age}"
+  end
```

Comment:
- [ ] When I create a teacher and try to list it instead of age it shows the specialization so please try to fix to match the video  also it would be better to remove this `optionsdfsdf` it makes it unprofessional
<img width="238" alt="Screenshot 2023-08-31 at 18 41 20" src="https://github.com/addisalemseifu/library-app-ruby/assets/61790587/c59a6d29-e40e-4e60-92ba-7a75e5effb86">

```suggestion
  persons.each do |person|
    puts "Name: #{person.name}, Age: #{person.age}"
  end
```